examples/harvest-recipe.yaml

```diff
@@ -0,0 +1,140 @@
+    '@type': RecipeFlow
+    skos:note: willow basket
+    action: use
+    recipeInputOf: bumblebee:53b44c35-d514-4945-b257-9f66c7ec0d67
+    involvesQuantity:
+      qudt:unit: unit:Number
+      qudt:numericValue: 2
+    temporalLength:
+      qudt:unit: unit:Hour
+      qudt:numericValue: 0.25
+
+  - '@id': bumblebee:76694411-781c-41b9-a265-5fe2ee456d79
+    '@type': RecipeFlow
+    skos:note: ladder
+    action: use
+    recipeInputOf: bumblebee:53b44c35-d514-4945-b257-9f66c7ec0d67
+    involvesQuantity:
+      qudt:unit: unit:Number
+      qudt:numericValue: 1
+    temporalLength:
+      qudt:unit: unit:Hour
+      qudt:numericValue: 0.25
+
+  - '@id': bumblebee:1178138b-21dd-42e1-8f43-c8462d14eaab
+    '@type': RecipeFlow
+    skos:note: person harvesting
+    action: work
+    recipeInputOf: bumblebee:53b44c35-d514-4945-b257-9f66c7ec0d67
+    involvesQuantity:
+      qudt:unit: unit:Number
+      qudt:numericValue: 2
+    temporalLength:
+      qudt:unit: unit:Hour
+      qudt:numericValue: 0.25
+
+  - '@id': bumblebee:11a2caee-a44d-4099-aa1b-56504684127f
+    '@type': Process
+    skos:note: ten people harvesting 30 apple trees in 3 hours
+
+  - '@id': bumblebee:60ac2d7c-e5cd-4de4-b981-0ad64ea716a2
+    '@type': Intent
+    skos:note: tree bearing apples
+    action: consume
+    inputOf: bumblebee:11a2caee-a44d-4099-aa1b-56504684127f
+    involvesQuantity:
+      qudt:unit: unit:Number
+      qudt:numericValue: 30
+
+  - '@id': bumblebee:0cdd34f8-e688-4d96-934e-b12b27d2c51b
+    '@type': Intent
+    skos:note: harvested apple tree
+    action: produce
+    outputOf: bumblebee:11a2caee-a44d-4099-aa1b-56504684127f
+    involvesQuantity:
+      qudt:unit: unit:Number
+      qudt:numericValue: 30
+
+  - '@id': bumblebee:b32fdc8f-dc39-4b9e-92e9-991aa5bac2ec
+    '@type': Intent
+    skos:note: apples
+    action: produce
+    outputOf: bumblebee:53b44c35-d514-4945-b257-9f66c7ec0d67
+    involvesQuantity:
+      qudt:unit: unit:Kilogram
+      qudt:numericValue: 600
+
+  - '@id': bumblebee:359dfdea-7f07-4977-820a-20214883068a
+    '@type': Intent
+    skos:note: willow basket
+    action: use
+    inputOf: bumblebee:11a2caee-a44d-4099-aa1b-56504684127f
+    involvesQuantity:
+      qudt:unit: unit:Number
+      qudt:numericValue: 10
+    temporalLength:
+      qudt:unit: unit:Hour
+      qudt:numericValue: 3
+
+  - '@id': bumblebee:32c8dc85-5700-406d-b1d0-f07c33210dc0
+    '@type': Intent
+    skos:note: ladder
+    action: use
+    inputOf: bumblebee:11a2caee-a44d-4099-aa1b-56504684127f
+    involvesQuantity:
+      qudt:unit: unit:Number
+      qudt:numericValue: 5
+    temporalLength:
+      qudt:unit: unit:Hour
+      qudt:numericValue: 3
+
+  - '@id': bumblebee:11fdbf89-90d4-47c8-a59d-37fdf2c3fb47
+    '@type': Intent
+    skos:note: person harvesting
+    action: work
+    inputOf: bumblebee:11a2caee-a44d-4099-aa1b-56504684127f
+    involvesQuantity:
+      qudt:unit: unit:Number
+      qudt:numericValue: 10
+    temporalLength:
```

Comment:
Does this mean that the range of involves would be expanded to include Agent? Or would there be some kind of "work availability" or "labor power" kind of EconomicResource? Or neither, just using that property for number of people not related to resources?

Reply:
Currently I don't see need for defining `Person` and `Resource` as disjoint. If we can in some situations consider dog, cat or a cow as a `Resource` I don't see why sometimes we can't act as both `Resource` and `Agent`. It may also come into play when modeling passenger transportation where process would *load* me in one place and *unload* me in another place, which differs from providing *load* of something unknown / undefined for me and providing *unload* of something unknown / undefined for me.

Reply:
As for the range we don't have to 'extend' it at all. All it does it implies that referenced thing also has this class. So one can use it with anything one doesn't problem with getting class `vf:EconomicResource` added by RDFS reasoner.

Reply:
One concern I have about this is that non-rdf implementations can't do any of that. So we would have to figure out how to translate that. Besides the concern I already expressed about "human resources" in corporate capitalism.... Not necessarily totally opposed, but seeing difficulties. And would need to see it carried through to commitments. But we don't need to do that now, since that is not the purpose of your use case....